pkg/listener/kafka/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM bitnami/minideb:jessie
+
```

Comment:
it'd be great if you make this a multi-stage container in which you build the kafka-consumer binary in the build step

Reply:
will do in separate PR (need to apply it into kubeless-controller's Dockerfile as well)